### ISSUE TITLE
@properties are suddenly filed as std:doc instead of std:attribute

### ISSUE THREAD

**Issue by flying-sheep** [sphinx-doc/sphinx#7060]
**Describe the bug**
I can’t link to my properties as ``:attr:`thing` `` anymore.
Inspecting the objects.inv, they now falsely appear under `std:doc` (/edit: also as `std:meth`, which is the real problem)

**To Reproduce**
Create an autosummary for a property:

```py
class Test:
    @property
    def thing():
        """Doc"""
```

```restructuredtext
.. autoclass:: Test
   :members:
```

**Environment info**
- OS: Arch Linux
- Python version: 3.8
- Sphinx version: 2.2.1
- Sphinx extensions: sphinx.ext.autodoc, sphinx.ext.autosummary

**Comment by tk0miya** [comment]
Now `:py:meth:` is used to refer properties. Could you try it please?

**Comment by flying-sheep** [comment]
I could but why was it changed? Properties are a transparent way to attach behavior to getting/setting attributes, that their whole point.

If it would stay that way, everyone would change all references whenever someone decides to go from attribute to property, where previously this was a transparent change for code and docs.

/edit: now that I know it’s `:meth:`, I found #6785. This one can be closed as #6785’s duplicate, but I agree with @zzzeek: please make it possible to refer to properties using `:attr:` again.
